README.md

```diff
@@ -12,3 +12,63 @@ The _RFC - Py-Spiffe features and use cases_ document is public at: https://docs
 * Gomez Coronel Andres (HPE)
 * Lambrecht Max (HPE)
 
+
+## Development setup
+
+### Prerequisites
+For basic development you will need:
+
+* Python 3.6
+* Pipenv (https://github.com/pypa/pipenv)
+* Pyenv (https://github.com/pyenv/pyenv)
+
+### Setup the environment
+
+Use `pyenv` to install the different version of Python.
+Python 3.6 is required for development and the other versions are required for testing.
+```
+pyenv install 3.6.x
+pyenv install 3.7.x
+pyenv install 3.8.x
+pyenv install 3.9.x
+```
+
+Clone the repository
+```
+git clone git@github.com:HewlettPackard/py-spiffe.git
```

Comment:
Not sure if this is the correct way, as we usually fork the repo and then clone our fork

Reply:
I used `git clone` because is the simplest scenario. 
Any one who use to fork the repos will fork it as usual so I think that there is no need to be very specific on it.